libs/designsystem/src/lib/components/modal/action-sheet/action-sheet.component.spec.ts

```diff
@@ -152,4 +155,20 @@ describe('ActionSheetComponent', () => {
       expect(cancelButton.nativeElement.innerText).toEqual(expected);
     });
   });
+
+  describe('cancel button color', () => {
+    it('should render with correct color', () => {
+      const cancelButton = fixture.debugElement.query(By.css('.cancel-btn'));
+      const element = cancelButton.nativeElement as HTMLElement;
+      expect(element).toHaveComputedStyle({
+        'border-width': '1px',
+        'border-style': 'solid',
+        'border-color': 'transparent',
+        'background-color': getColor('white', 'contrast'),
+        /* prettier-ignore */
+        // tslint:disable-next-line: prettier
```

Comment:
```suggestion
```
I'm a bit unsure - but i think just having the `/* prettier-ignore */` comment is sufficient to disable prettier for that line